src/Storages/MergeTree/MergeTreeRangeReader.cpp

```diff
@@ -905,7 +905,9 @@ void MergeTreeRangeReader::executePrewhereActionsAndFilterColumns(ReadResult & r
             if (prewhere->remove_prewhere_column)
                 result.columns.erase(result.columns.begin() + prewhere_column_pos);
             else
-                result.columns[prewhere_column_pos] = DataTypeUInt8().createColumnConst(result.num_rows, 1u)->convertToFullColumnIfConst();
```

Comment:
Ok.

Just in case I want to make sure that we have run the search for
`DataTypeUInt8().createColumn` in all project files...